Yoast/wordpress-seo
# P1-190 add ids

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Refactors IDs to use location.

## Relevant technical choices:

* Removed the unique ID because we can not reliably know what it will be. We should ensure the ID to be unique ourselves.

## Test instructions

This PR can be tested by following these steps:

* See test instructions on premium: https://github.com/Yoast/wordpress-seo-premium/pull/3022
* Only the social previews would need to be checked separately in free, as those are different.

## UI changes

## Documentation

## Quality assurance


Fixes https://yoast.atlassian.net/browse/P1-190
